js/main.js

```diff
@@ -0,0 +1,345 @@
+  linkToLive: 'https://github.com/solorics',
+  btn: 'See Source',
+  side: ['Next Project', 'Previous Project'],
+  icons: ['./assets/icons/Vector.png', './assets/icons/Union.png'],
+  btnForPopup: ['See Live', 'See Source'],
+  nextPrevious: ['./assets/icons/pre.png', './assets/icons/next.png'],
+}, {
+  id: '6',
+  class: 'item10',
+  title: 'Project name goes here',
+  features: ['HTML/CSS', 'Ruby on Rails', 'JavaScript'],
+  image: './assets/images/Rectangle26.svg',
+  img: './assets/images/Union.svg',
+  sideArrows: ['./assets/icons/left.png', './assets/icons/right.png'],
+  sideImage: [
+    './assets/images/Rectangle26.svg',
+    './assets/images/Rectangle26.svg',
+    './assets/images/Rectangle26.svg',
+    './assets/images/Rectangle26.svg',
+  ],
+  description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+enim ad minim veniam, quis nostrud exercitation ullamco laboris
+<br /><br />
+nisi Ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit
+amet, consectetur adipiscing elit, sed do eiusmod tempor
+incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+veniam, quis nostrud exercitation ullamco laboris nisi.`,
+  linkToSource: 'https://solorics.github.io/Portfolio-Mobile-FIrst/',
+  linkToLive: 'https://github.com/solorics',
+  btn: 'See Source',
+  side: ['Next Project', 'Previous Project'],
+  icons: ['./assets/icons/Vector.png', './assets/icons/Union.png'],
+  btnForPopup: ['See Live', 'See Source'],
+  nextPrevious: ['./assets/icons/pre.png', './assets/icons/next.png'],
+},
+];
+
+const projectList = document.querySelector('#portfolio ul.grid');
+
+function loadProjects() {
+  for (let i = 0; i < projects.length; i += 1) {
+    const projectItemtemplate = `
+    <li class="project-item ${projects[i].class}">
+      <img
+        class="project-img"
+        src="${projects[i].image}"
+        alt="laptop"
+      />
+      <div class="container container-information hover">
+        <h2 class="heading">${projects[i].title}</h2>
+        <ul class="features">
+          <li class="feature">${projects[i].features[0]}</li>
+          <li class="feature">${projects[i].features[1]}</li>
+          <li class="feature">${projects[i].features[2]}</li>
+        </ul>
+        <button data-id="${projects[i].id}" class="c-projects">
+          See this project &rarr;
+        </button>
+      </div>
+    </li>
+  `;
+
+    projectList.innerHTML += projectItemtemplate;
+  }
+}
+
+window.addEventListener('load', loadProjects());
+
+function createModal(id) {
+  const pCardTemplate = `
+  <div class="overlay hidden">
+      <article class="modal container">
+        <div class="modal__header">
+          <div class="modal__header_title flex space-between">
+            <h1 class="top_title">${projects[id].title}</h1>
+            <button type="button" class="top_btn">
+              &#x2716
+            </button>
```

Comment:
- [ ] Kindly check the font-size, font-weight position, other dimensions of the popup title and ensure it fits the figma design since there are some slight differences. 
- [ ] Please the close button seems to dissappear, kindly check and fix the position. 

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-19 23-14-25](https://user-images.githubusercontent.com/65233860/202875426-26071785-3d1b-4834-a7db-1110ae1e2ede.png)  |  ![Screenshot from 2022-11-19 23-14-01](https://user-images.githubusercontent.com/65233860/202875434-e3047880-352b-4b75-8ac3-a79417a1104f.png)

Reply:
- [ ] As suggested by the previous reviewer, Kindly check the font-weight and font family, follow as shown in the image below with highlighted font values:

<img width="1488" alt="Screenshot 2022-11-22 at 1 12 50 AM" src="https://user-images.githubusercontent.com/70269507/203168672-e9be16a9-d7aa-4461-92ba-033387ae3166.png">

- [ ] Kindly make sure close button is responsive and fits to mobile screen currently it seems to disappear, kindly check and fix the position.

<img width="617" alt="Screenshot 2022-11-22 at 1 11 58 AM" src="https://user-images.githubusercontent.com/70269507/203168626-2838a870-3a85-4542-9c79-67bfbd9ef12e.png">